nginx-directory/deployment.libsonnet

```diff
@@ -0,0 +1,29 @@
+{
+  local container = $.core.v1.container,
+
+  nginx_container::
+    container.new('nginx', $._images.nginx) +
+    container.withPorts($.core.v1.containerPort.new('http', 80)) +
+    $.util.resourcesRequests('50m', '100Mi'),
+
+  local deployment = $.apps.v1.deployment,
+
+  nginx_deployment:
+    deployment.new('nginx', 1, [$.nginx_container]) +
+    $.util.configMapVolumeMount($.nginx_config_map, '/etc/nginx') +
```

Comment:
Where is nginx_config_map in nginx-directory?

Reply:
in `configfile.libsonnet`